scrapers/Oldje.yml

```diff
@@ -0,0 +1,53 @@
+name: Oldje
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - oldje.com/video
+    scraper: oldjeScraper
+  - action: scrapeXPath
+    url:
+      - oldje-3some.com/videos
+    scraper: oldje3someScraper
+xPathScrapers:
+  oldjeScraper:
+    scene:
+      Studio:
+         Name:
+           fixed: Oldje
+      Title:
+        selector: //h1[@class='inlay_title']
```

Comment:
the older scenes seem to have a different layout eg `https://www.oldje.com/video/lolita-taylor/dirty-santa`
```yaml
        selector: //h1[@class='inlay_title']|//h1[@class="preview_header"]/span[@class="bold"]
```